### PR TITLE
New version: Agilent.LabAdvisor version 2.19.503

### PR DESCRIPTION
**Pull request by Spacefish**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/67567)

### CODE REVIEW

File: manifests/a/Agilent/LabAdvisor/2.19.503/Agilent.LabAdvisor.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+Publisher: Agilent Technologies
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+Author: Agilent
+PackageName: Agilent Lab Advisor
+PackageUrl: https://www.agilent.com/en/product/liquid-chromatography/hplc-ce-software/instrument-control-software/lab-advisor-software
+License: Copyright (c) Agilent Technologies
+LicenseUrl: https://labadvisor.pl29.agilent.com/Software%20License%20Terms.txt
+# Copyright:
```
Comment (Trenly) [inline manifests/a/Agilent/LabAdvisor/2.19.503/Agilent.LabAdvisor.locale.en-US.yaml:16]:
```suggestion
License: Proprietary
LicenseUrl: https://labadvisor.pl29.agilent.com/Software%20License%20Terms.txt
Copyright: Copyright (c) Agilent Technologies
```

**Review by Trenly — CHANGES_REQUESTED** [review]
A copyright is not a license

File: manifests/a/Agilent/LabAdvisor/2.19.503/Agilent.LabAdvisor.locale.en-US.yaml (hunk 1 above)
Comment (Spacefish) [reply manifests/a/Agilent/LabAdvisor/2.19.503/Agilent.LabAdvisor.locale.en-US.yaml]: Yes you are right, thanks for the change!

File: manifests/a/Agilent/LabAdvisor/2.19.503/Agilent.LabAdvisor.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+Installers:
+- Architecture: x64
+  InstallerUrl: https://labadvisor.pl29.agilent.com/AgilentLabAdvisor_2.19_SR1.503.msi
+  InstallerSha256: 8576c67bc4189fb727035fb3ff87a7978c67c5bc651aab7a2f8aafa4d5e0f222
+  ProductCode: '{B64A00F5-D3FF-4683-AAB7-84B62EE8FBF1}'
```
Comment (ItzLevvie) [inline manifests/a/Agilent/LabAdvisor/2.19.503/Agilent.LabAdvisor.installer.yaml:15]:
```suggestion
  ProductCode: '{631D82BE-8FB8-4B7C-9E61-B7DFFBA30601}'
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/a/Agilent/LabAdvisor/2.19.503/Agilent.LabAdvisor.installer.yaml (hunk 2 above)
Comment (Spacefish) [reply manifests/a/Agilent/LabAdvisor/2.19.503/Agilent.LabAdvisor.installer.yaml:15]: Yes that´s true, i should have used the script :(

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**